tests/rados/pool_tests.py

```diff
@@ -400,3 +400,43 @@ def run(ceph_cluster, **kw):
             log.info(f"Completed the test of pg_min_num on pool: {entry['pool_name']} ")
         log.info("pg_min_num tests completed")
         return 0
+
+    if config.get("verify_pg_num_limit"):
+        """
+        This test is to verify the pg number limit for a pools.
+        Script cover the following steps-
+           1. Creating a pool with default pg number
+           2. Set the pg number more than the limit that is more than 128
+           3. Cheking that pg number getting reduced or not
+        """
+
+        pool_name = config.get("verify_pg_num_limit")["pool_name"]
+        rados_obj.create_pool(pool_name)
+
+        # set the pg number more than 128
+        cmd = f"ceph osd pool set {pool_name} pg_num 256"
```

Comment:
If it takes quite a long time to reduce from 256 -> 128. can we keep a lower count, say 150? The goal of the test is to check if the PGs get below 128, i.e. the threshold level.

Reply:
Yes, I agree that and I will change the code.